.github/workflows/pull-request-next.yml

```diff
@@ -37,3 +41,20 @@ jobs:
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
       - run: yarn install
       - run: yarn lerna run test:functional
+  visual:
+    name: Visual Tests
```

Comment:
Other jobs do not have names (they use job id as name, eg `functional`). Please remove.